pkgs/development/python-modules/atpublic/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+# python3 only
+buildPythonPackage rec {
+  pname = "atpublic";
+  version = "1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0i3sbxkdlbb4560rrlmwwd5y4ps7k73lp4d8wnmd7ag9k426gjkx";
+  };
+
+  # TypeError: calling <function public at 0x7ffff26dbb70> returned None, not a test
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    homepage = "https://public.readthedocs.io/en/latest/";
+    description = "@public for populating __all__";
+    longDescription = ''
+      This is a very simple decorator and function which populates a module's
+      __all__ and optionally the module globals.
+
+      This provides both a pure-Python implementation and a C implementation.  It is
+      proposed that the C implementation be added to builtins_ for Python 3.6.
```

Comment:
Has the proposal been adopted?

Reply:
It looks like it might've been rejected? https://bugs.python.org/issue26632